code/modules/food_and_drinks/kitchen_machinery/cooker.dm

```diff
@@ -179,29 +179,29 @@
 	var/obj/item/food/type = new(get_turf(src))
 	return type
 
-/obj/machinery/cooker/attackby__legacy__attackchain(obj/item/I, mob/user, params)
+/obj/machinery/cooker/item_interaction(mob/living/user, obj/item/used, list/modifiers)
```

Comment:
Multiple returns here that should be `return ITEM_INTERACT_COMPLETE`

Reply:
Fixed